client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import NavItem from 'components/section-nav/item';
+import NavTabs from 'components/section-nav/tabs';
+import Search from 'components/search';
+import SectionNav from 'components/section-nav';
+
+const DEFAULT_STATUS = 'pending';
+
+export class CommentNavigation extends Component {
+	getStatusPath = status => ( status && DEFAULT_STATUS !== status )
+		? `/comments/${ status }/${ this.props.siteSlug }`
+		: `/comments/${ this.props.siteSlug }`;
+
+	isSelectedStatus = status => DEFAULT_STATUS === status
+		? ! this.props.status || status === this.props.status
```

Comment:
I think we can avoid the ternary here by updating the default status to point at `pending`
```jsx
status defaultProps = {
   status: 'pending',
}
```